<launch>
    <arg name="id" default="$(env uav_id)" doc="uav_id type [1, 2, 3]"/>

    <node pkg="radio_proxy" type="px4_proxy" name="px4_proxy" output="screen">
        <param name="id" value="$(arg id)"/>
        <!-- <param name="port" value="/dev/usb_radio" /> -->
        <param name="port" value="/dev/ttyUSB1" />
        <param name="baud_rate" value="230400" />
    </node>
</launch>